src/utils/calendar.ts

```diff
@@ -0,0 +1,44 @@
+import ical from "ical-generator";
+
+import { VenueEvent } from "types/venues";
+
+import { eventEndTime, eventStartTime } from "utils/event";
+
+import { WithVenueId } from "utils/id";
+
+import { getFullVenueInsideUrl } from "utils/url";
+
+export interface CreateCalendarProps {
+  calendarName: string;
+  events: WithVenueId<VenueEvent>[];
+}
+
+const createCalendar = ({ calendarName, events }: CreateCalendarProps) => {
```

Comment:
We'll want to replace the `???` with an explicit type here (whatever type `cal` is)

```suggestion
const createCalendar = ({ calendarName, events }: CreateCalendarProps): ??? => {
```

Reply:
We'll still need to replace this with the actual correct type